### ISSUE TITLE
VP9 Codec has been disabled for videos under 1440p can we please have this fixed this is the main reason why I haven't updated

### ISSUE THREAD

**Issue by SuperAmirhamza** [arichorn/uYouEnhanced#265]
![image](https://github.com/arichorn/uYouEnhanced/assets/45406543/7e7428b9-13fc-4661-9399-82aa2563c5f6)

**Comment by KieranBLK** [comment]
> ![image](https://github.com/arichorn/uYouEnhanced/assets/45406543/7e7428b9-13fc-4661-9399-82aa2563c5f6)

That's a YouTube issue. YouTube has never allowed VP9 on qualities under 1440p, it's always been AV1 for those.

**Comment by SuperAmirhamza** [comment]
> > ![image](https://github.com/arichorn/uYouEnhanced/assets/45406543/7e7428b9-13fc-4661-9399-82aa2563c5f6)
> 
> That's a YouTube issue. YouTube has never allowed VP9 on qualities under 1440p, it's always been AV1 for those. 

Yeah it's who created the YTUHD tweak he made it so quality under 1440p isn't vp9

**Comment by KieranBLK** [comment]
> > > ![image](https://github.com/arichorn/uYouEnhanced/assets/45406543/7e7428b9-13fc-4661-9399-82aa2563c5f6)
> > 
> > That's a YouTube issue. YouTube has never allowed VP9 on qualities under 1440p, it's always been AV1 for those. 
> 
> Yeah it's who created the YTUHD tweak he made it so quality under 1440p isn't vp9

They didn't. Like I said, it's a stock YT behavior. YouTube has never allowed VP9 on any quality other than 1440p and above, you can check that on the App Store app and the website, it's also best to not permanently force VP9 for all qualities considering that the sideloaded app doesn't have hardware encoder access and can't sustain proper performance and efficiency while streaming it for too long.
